internal/server/deputy_hub.go

```diff
@@ -0,0 +1,41 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/sirius"
+)
+
+type DeputyHubInformation interface {
+}
+
+type deputyHubVars struct {
+	Path      string
+	XSRFToken string
+	Error     string
+	Errors    sirius.ValidationErrors
+}
+
+func loggingInfoForDeputyHub(client DeputyHubInformation, tmpl Template) Handler {
```

Comment:
Not a fan of the naming of this function (I know this follows the pattern in Workflow) as it isn't logging anything, it is rendering a template.